code/game/world.dm

```diff
@@ -220,4 +220,14 @@ var/world_topic_spam_protect_time = world.timeofday
 /world/proc/SetupExternalRSC()
 	if(!CONFIG_GET(string/resource_url))
 		return
-	GLOB.external_rsc_url = CONFIG_GET(string/resource_url)
\ No newline at end of file
+	GLOB.external_rsc_url = CONFIG_GET(string/resource_url)
+
+
+/world/proc/update_hub_visibility(new_visibility)
+	if(new_visibility == GLOB.hub_visibility)
+		return
+	GLOB.hub_visibility = new_visibility
+	if(GLOB.hub_visibility)
+		hub_password = "kMZy3U5jJHSiBQjr"
```

Comment:
shouldnt this config get the hub password

Reply:
The hub password won't change so there really isn't harm in hardcoding it, though if you want I can config it back.